### PR TITLE
Rails 5

### PR DESCRIPTION
**Pull request by asurin**
@pmoore @dezmathio

### CODE REVIEW

File: lib/contentful_rails/engine.rb [hunk 1]
```diff
@@ -59,9 +59,15 @@ class Engine < ::Rails::Engine
...
       Mime::Type.register content_type, :contentful_json, [content_type]
-
-      ActionDispatch::ParamsParser::DEFAULT_PARSERS[Mime::Type.lookup(content_type)] = lambda do |body|
-        JSON.parse(body)
+      # Rails 4
+      if ActionDispatch::ParamsParser.constants.include? :DEFAULT_PARSERS
+        ActionDispatch::ParamsParser::DEFAULT_PARSERS[Mime::Type.lookup(content_type)] = lambda do |body|
+          JSON.parse(body)
+        end
+      else # Rails 5
```
Comment (dezmathio) [inline lib/contentful_rails/engine.rb]: Is this a deprecation from 4 -> 5 ? Seems good to me

Comment (asurin) [reply lib/contentful_rails/engine.rb]: Refactor, they didn't even deprecate it just moved it.  Similarly the raise is necessary to restore rails 4 behavior on the skip...

File: lib/contentful_rails/engine.rb [hunk 2]
```diff
@@ -59,9 +59,15 @@ class Engine < ::Rails::Engine
...
+        ActionDispatch::Http::Parameters::DEFAULT_PARSERS[Mime::Type.lookup(content_type)] = lambda do |body|
+          JSON.parse(body)
+        end
```
Comment (pmoore) [inline lib/contentful_rails/engine.rb]: fyi @prosanelli has a PR open from a couple days ago for this change.   It manually checks the Rails version, but is a little drier :p
https://github.com/weddingwire/contentful_rails/pull/5/files

**Comment by asurin** [comment]
Moved over @prosanelli's commit as I also needed the filter option change to get the app to boot.  This teaches me to check PRs first lol

**Review by pmoore — APPROVED** [review]

**Pull request merged**